roles/infrastructure/defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Infrastructure Defaults
+infrastructure_region: eastus
+infrastructure_tags: []
+
+# Networking
+infrastructure_nsg_name: aap-ports
+infrastructure_vnet_cidr: 172.16.0.0/22
+infrastructure_vnet_name: aap
+infrastructure_vnet_subnets:
+  - name: controller-eda-hub
```

Comment:
Shouldn't we create them separated as AWS? https://github.com/ansible-content-lab/aws_ansible_deployment/blob/main/roles/infrastructure/defaults/main.yml#L8-L20

Reply:
Ran it by Scott. Doing that needed 2046 IPs (/21) to keep with the same /24 breakup since AWS was missing a subnet for the db there. Still too much, in my opinion, with a /22 here when this could be down with /26 with plenty of room to spare for most deployments.